Formula/scio.rb

```diff
@@ -0,0 +1,26 @@
+class Scio< Formula
+  desc "Scala API for Google Cloud Dataflow and Apache Beam"
+  homepage "https://github.com/spotify/scio/"
+  url "https://github.com/spotify/scio/releases/download/v0.2.0/scio-repl-0.2.0.jar"
+  sha256 "220e03dfd866d8475fc7943f60376ea4fe0862c3520674e46ef8262a09d1756c"
+
+  bottle :unneeded
+
+  def install
+    libexec.install "scio-repl-#{version}.jar"
+    (bin+"scio-repl").write <<-EOS.undent
+      #!/bin/bash
+      java_opts=()
+      scio_opts=()
+      for o in "$@"; do
+          if [[ "$o" =~ ^-D.* ]]; then
+              java_opts+=("$o")
+          else
+              scio_opts+=("$o")
+          fi
+      done
```

Comment:
Please submit this wrapper script logic to the upstream project, it's too complicated to live in a Homebrew formula.